samples/winapi_filefind/main.c

```diff
@@ -10,12 +10,15 @@ int main(void)
 {
     XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
 
+    // Create a variable for WinAPI error checking
+    DWORD error = GetLastError();
```

Comment:
Is there a specific reason why the variable gets immediately assigned the "LastError" without it being used/checked? Wouldn't it be possible to just write `DWORD error;`? (Maybe I'm missing something, in that case, enlighten me please 🤓)